set_program_name(domain) requires newer py-gtk2, but is kept back
due to implied py-Nummeric update.

$OpenBSD: patch-system-config-printer_py,v 1.5 2008/10/27 13:29:13 ajacoutot Exp $
--- system-config-printer.py.orig	Tue Oct 21 11:02:45 2008
+++ system-config-printer.py	Mon Oct 27 14:30:20 2008
@@ -333,7 +333,7 @@ class GUI(GtkGUI, monitor.Watcher):
         np.NewPrinterWindow.set_transient_for(self.MainWindow)
 
         # Set up "About" dialog
-        self.AboutDialog.set_program_name(domain)
+        #self.AboutDialog.set_program_name(domain)
         self.AboutDialog.set_version(config.VERSION)
         self.AboutDialog.set_icon_name('printer')
 
@@ -377,7 +377,7 @@ class GUI(GtkGUI, monitor.Watcher):
         self.dests_iconview.set_item_width (145)
         self.dests_iconview.set_pixbuf_column (1)
         self.dests_iconview.set_text_column (2)
-        self.dests_iconview.set_tooltip_column (3)
+        #self.dests_iconview.set_tooltip_column (3)
         self.dests_iconview.connect ('item-activated',
                                      self.dests_iconview_item_activated)
         self.dests_iconview.connect ('selection-changed',
@@ -3707,7 +3707,7 @@ class NewPrinterGUI(GtkGUI):
         # Try to get make and model via SNMP
         if host:
             os.environ["HOST"] = host
-            cmd = '/usr/lib/cups/backend/snmp "${HOST}"'
+            cmd = '${LOCALBASE}/libexec/cups/backend/snmp "${HOST}"'
             debugprint (host + ": " + cmd)
             stdout = None
             try:
@@ -5098,7 +5098,7 @@ class NewPrinterGUI(GtkGUI):
                 try:
                     # We want this to be in the current natural language,
                     # so we intentionally don't set LC_ALL=C here.
-                    p = subprocess.Popen (['/usr/bin/cupstestppd',
+                    p = subprocess.Popen (['${LOCALBASE}/bin/cupstestppd',
                                            '-rvv', filename],
                                           stdin=file("/dev/null"),
                                           stdout=subprocess.PIPE,
